recipes/graylog-forwarder/files/post-install

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+set -e
+
+user="graylog-forwarder"
+group="graylog-forwarder"
+datadir="/usr/share/graylog-forwarder"
+cfgdir="/etc/graylog/forwarder"
+journaldir="/var/lib/graylog-forwarder"
+
+# Flag to indicate if this is an upgrade
+upgrade="false"
+
+case "$1" in
+	# DEB based systems
+	configure)
+		if [ -z "$2" ]; then
+			upgrade="false"
+
+		else
+			upgrade="true"
+		fi
+		;;
+	abort-deconfigure|abort-upgrade|abort-remove)
+		;;
+	# RPM based systems
+	1)
+		# Installation
+		upgrade="false"
+		;;
+	2)
+		# Upgrade
+		upgrade="true"
+		;;
+	*)
+		echo "[ERROR] post-install script called with unknown argument: '$1'"
+		exit 1
+		;;
+esac
+
+#ensure files are owned by graylog-forwarder user
+chown -R "$user:$group" "$datadir"
```

Comment:
We already create the `datadir` in `pre-install`. So once that directory is not `/usr/share/graylog-forwarder` anymore, we can remove this.

Reply:
I found that without this line, all the files laid down are owned by root.

Reply:
This is fine for the `/usr/share/graylog-forwarder` directory. The forwarder doesn't need to write to this directory.